### DISCUSSION TITLE
Error accessing this tool

### DISCUSSION THREAD

**Discussion by ac72026** [prisma/prisma#13336] [Q&A]
Just curious if this tool is live for me or am I facing an error on my side. When I click on connect DB it gives me an error
<img width="2534" alt="Screen Shot 2022-05-12 at 10 39 34 AM" src="https://user-images.githubusercontent.com/25291766/168136176-5e1571e2-fab6-43e1-936d-af31d07a9b4f.png">
.

**Comment by austincrim** [comment]
Hey @sebass-surfer 👋,

It would be great if you could send a description of your issue and the error message on Intercom (that chat box in the bottom right of your screenshot)!

**Reply by ac72026** [reply]
<img width="2557" alt="Screen Shot 2022-05-13 at 7 47 10 AM" src="https://user-images.githubusercontent.com/25291766/168308921-b512a8d7-f940-439e-a8ce-5fbbc989fd39.png">

Application error: a client-side exception has occurred (see the browser console for more information).

**Reply by pantharshit00** [reply]
Hello @sebass-surfer 

Are you still experiencing this issue? Please let us know so that we can close this discussion.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
